hs_access_control/models/resource.py

```diff
@@ -227,6 +410,39 @@ def __get_raw_group_privilege(self, this_user):
             response2 = PrivilegeCodes.NONE
         return response2
 
+    def __get_raw_community_privilege(self, this_user):
```

Comment:
What if `this_user` is NOT active?

Reply:
You won't get any privilege so it'll return PrivilegeCodes.NONE